backend/apps/audit/signals.py

```
@@ -0,0 +1,113 @@
+    from apps.core.models import AuditableModel
+
+    if sender is AuditEvent:
+        return False
+    return issubclass(sender, AuditableModel)
+
+
+def _model_snapshot(instance) -> dict:
+    """Serialise instance fields to a plain dict (JSON-safe)."""
+    try:
+        return model_to_dict(instance)
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)